Mutagen.Bethesda.Analyzers.Skyrim/Mutagen.Bethesda.Analyzers.Skyrim.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=extensions/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
This marks the extensions folder to be no namespace provider, meaning the namespace will not automatically be set up to follow the folder structure with `namespace Mutagen.Bethesda.Analyzers.Skyrim.Extensions;` and instead it will stay at `namespace Mutagen.Bethesda.Analyzers.Skyrim;` meaning all namespaces in the project can use the extensions without adding any `using` statements.

Reply:
Hmm, I dont necessarily mind the extensions living in their own namespace to match the folder they're in.   IDEs will import them easily enough.    If we dont want to have to do that, I think I'd prefer adding implicit namespaces to the project instead?  Similar result, but without a Jetbrains specific file

https://devblogs.microsoft.com/dotnet/welcome-to-csharp-10/